CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Grafana Mimir
 
 * [CHANGE] Increased default configuration for `-server.grpc-max-recv-msg-size-bytes` and `-server.grpc-max-send-msg-size-bytes` from 4MB to 100MB. #1883
+* [CHANGE] Default values have changed for the following settings: #1909
```

Comment:
I would slightly explain the explanation saying that we're doing this change to improve query performance using the default config. The trade-off is that the last 12h of data is always queried only from ingesters and never from store-gateways.